ase/scripts/run_in_ase

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+# Run a binary using the ASE simulation library instead of the standard libopae-c.
+env LD_PRELOAD="libopae-c-ase.so${LD_PRELOAD:+:$LD_PRELOAD}" "$@"
```

Comment:
Huh. I think I've only seen `LD_PRELOAD` as being space-delimited, not `:`-delimited, but I guess the dynamic linker accepts either.